app/src/main/java/com/android/sample/ui/listActivities/LikedActivities.kt

```diff
@@ -82,7 +86,14 @@ fun LikedActivitiesScreen(
             selectedItem = navigationActions.currentRoute())
       }) { paddingValues ->
         Box(modifier = modifier.fillMaxSize().padding(paddingValues)) {
-          val likedActivitiesList by remember { mutableStateOf(profile?.likedActivities) }
```

Comment:
added offline support by integrating Room to fetch liked activities from the local database when the network is unavailable, enhancing the app's functionality in offline mode.